src/redux/ui/screens/input-data/state.ts

```diff
@@ -1,17 +1,28 @@
-export interface InputDataValidator {
-    id: string;
-    name: string;
-    icon?: string;
-    website?: string;
+export interface IScreenInputState {
+    // 'walletPubKey-blockchain-chainId-address-step-tab'
+    [screenKey: string]: IScreenInputData;
 }
 
 export interface IScreenInputData {
-    // key: 'walletPubKey-blockchain-chainId-address-tab'
-    [key: string]: {
-        validators: InputDataValidator[];
+    validation: IScreenInputDataValidations;
+
+    // Save here any input data
+    data: {
+        [key: string]: any;
     };
 }
 
-export interface IScreenInputState {
-    [screen: string]: IScreenInputData;
+export interface IScreenInputDataValidations {
+    fieldsErrors: {
```

Comment:
something is missing here :) there is no info about the field name
Transform this into:
```ts
fieldsErrors: {
  [fieldName: string]: {
    type: 'ERROR_MSG' | 'WARN_MSG';
    message: string;
  }[];
}